core/client/assets/sass/layouts/settings.scss

```diff
@@ -394,18 +394,29 @@
     display: flex;
 
     @media (max-width: 600px) {
+        position: relative;
         margin-bottom: 20px;
     }
     @media (min-width: 601px) {
         margin-bottom: 10px;
+
+        &.last-navigation-item {
```

Comment:
Why don't you use the `:last-child` pseudo-class to style / jquery select this instead of having to add another class? I think it'd clean up the javascript a bit